app/src/main/java/org/kiwix/kiwixmobile/main/MainActivity.java

```diff
@@ -235,6 +235,10 @@ public void onFinish() {
   private AppCompatButton downloadBookButton;
   private ActionBar actionBar;
   private TextView tabSwitcherIcon;
+  private FileReader fileReader;
```

Comment:
I get a lint warning on this line saying **Field can be converted to a local variable**. Do not ignore lint warning for the piece of code with which you are dealing. You can run lint from **Analyze -> Inspect Code**.